spec/requests/mentees_request_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "rails_helper"
+
+RSpec.describe "Mentees", type: :request do
+  headers = nil
+  before do
+    user = FactoryBot.create(:user)
+    post "/login", params: { "email" => user.email, "password" => user.password }
+    user_info = JSON.parse(response.body)
+    headers = { "Authorization" => "Bearer " + user_info["token"] }
+  end
+  describe "GET /index" do
+    it "returns empty list of mentee" do
```

Comment:
This test description does not match the actual test. Maybe you meant to write two tests here– one returns an empty list and the other returns a mentee?